phpunit/functional/Glpi/Form/QuestionType/QuestionTypeRequesterTest.php

```diff
@@ -0,0 +1,159 @@
+{
+    use FormTesterTrait;
+
+    public static function singleRequesterAnswerIsDisplayedInTicketDescriptionProvider(): iterable
+    {
+        $glpi_id = getItemByTypeName(User::class, "glpi", true);
+        yield 'simple user' => [
+            'answer' => ["users_id-$glpi_id"],
+            'expected' => "glpi",
+        ];
+
+        $test_group_1_id = getItemByTypeName(Group::class, "_test_group_1", true);
+        yield 'simple group' => [
+            'answer' => ["groups_id-$test_group_1_id"],
+            'expected' => "_test_group_1",
+        ];
+    }
+
+    #[DataProvider("singleRequesterAnswerIsDisplayedInTicketDescriptionProvider")]
+    public function testSingleRequesterAnswerIsDisplayedInTicketDescription(
+        array $answer,
+        string $expected
+    ): void {
+        $builder = new FormBuilder();
+        $builder->addQuestion("Requester", QuestionTypeRequester::class);
+        $builder->addDestination(FormDestinationTicket::class, "My ticket");
+        $form = $this->createForm($builder);
+
+        $ticket = $this->sendFormAndGetCreatedTicket($form, [
+            "Requester" => $answer,
+        ]);
+
+        $this->assertStringContainsString(
+            "1) Requester: $expected",
+            strip_tags($ticket->fields['content']),
+        );
+    }
+
+    public static function multipleRequesterAnswerIsDisplayedInTicketDescriptionProvider(): iterable
+    {
+        $glpi_id = getItemByTypeName(User::class, "glpi", true);
+        $tech_id = getItemByTypeName(User::class, "tech", true);
+        yield 'multiple users' => [
+            'answer' => [
+                "users_id-$glpi_id",
+                "users_id-$tech_id",
+            ],
+            'expected' => "glpi, tech",
+        ];
+
+        $test_group_1_id = getItemByTypeName(Group::class, "_test_group_1", true);
+        $test_group_2_id = getItemByTypeName(Group::class, "_test_group_2", true);
+        yield 'multiple groups' => [
+            'answer' => [
+                "groups_id-$test_group_1_id",
+                "groups_id-$test_group_2_id",
+            ],
+            'expected' => "_test_group_1, _test_group_2",
+        ];
+    }
+
+    #[DataProvider("multipleRequesterAnswerIsDisplayedInTicketDescriptionProvider")]
+    public function testMultipleRequesterAnswerIsDisplayedInTicketDescription(
+        array $answer,
+        string $expected
+    ): void {
+        $builder = new FormBuilder();
+        $builder->addQuestion("Requester", QuestionTypeRequester::class);
+        $builder->addDestination(FormDestinationTicket::class, "My ticket", ['is_multiple_actors' => true]);
+        $form = $this->createForm($builder);
+
+        $ticket = $this->sendFormAndGetCreatedTicket($form, [
+            "Requester" => $answer,
+        ]);
+
+        $this->assertStringContainsString(
+            "1) Requester: $expected",
+            strip_tags($ticket->fields['content']),
+        );
+    }
```

Comment:
It seems to be the exact same code as the `testSingleRequesterAnswerIsDisplayedInTicketDescription` method. I guess you can merge the two providers and use a unique test method. You should check the Assignee and Observer questions too.